### ISSUE TITLE
phoronix-test-suite.php Bug

### ISSUE THREAD

**Issue by inewton888** [phoronix-test-suite/phoronix-test-suite#407]
I am installing phoronix on a new install of 64bit Windows 10 Pro on a Xeon workstation.  Installation went fine with phoronix-test-suite.bat except when installation progresses to this line in phoronix-test-suite.php

pts_client::user_agreement_check($sent_command);

I get an error message that says "This app can't run on your PC. To find a version for your PC, check with the software publisher."  After clicking "OK," I see "Access is denied" at the command prompt.  The rest of the PHP script completes.

**Comment by michaellarabel** [comment]
Hmmm haven't ran into this issue myself, perhaps are you running on some elevated security settings or so?

**Comment by michaellarabel** [comment]
Also, does PTS work if you run it from within the phoronix-test-suite directory as opposed to first running the .bat installer?

**Comment by inewton888** [comment]
Security settings are left default in Windows 10.  I just tried executing phoronix-test-suite.php after turning off Antivirus... received the same error messages.  Running tests from within phoronix-test-suite directory seem to work other than Did Not Produce Result or Non-zero exit status when running pts/cpu.  Running pts/memory worked but path problem when trying to launch OpenBenchMarking.
Launch OpenBenchMarking.org - system can't find the path specified.

**Comment by inewton888** [comment]
Btw using phoronix-test-suite 9.2.1

**Comment by michaellarabel** [comment]
It's possible when running pts/cpu it's triggering some oddities, some tests within that test suite aren't designed to run on Windows but it should be ignoring them. Will take a look next time i am on Windows. For the did not produce result / non-zero exit status, some tests are finicky on Windows depending upon environment.

**Comment by inewton888** [comment]
Ah I C.  Trying to compare results I got using Phoronix in Centos vs. Win 10 to figure out if there is a hardware issue or a driver issue with the workstation in question.  It may seem that the user agreement function doesn't impact the running of tests or does it?

**Comment by michaellarabel** [comment]
No that function should have no impact on the functionality of the program.

**Comment by inewton888** [comment]
The second time pts/cpu reported with problem message "This app can't run on your PC. To find a version for your PC, check with the software publisher" is to do with p7zip below...

RUNNING: cygwin-setup-x86_64.exe -q -P p7zip -WAccess is denied.

I also tried downloading and installing 64bit p7zip and reran pts/cpu receiving the same problem messages. This bug should probably have its own issue number.

**Comment by inewton888** [comment]
OK putting cygwin-setup-x86_64.exe in phoronix-test-suite directory did the trick to fix pts/cpu.  Also I had to reinstall the tests in that suite to get pts/cpu working.
**The problem:**  When running phoronix-test-suite.bat, Cygwin failed to setup, so I downloaded it and installed manually.  So what happened was probably pts/cpu installation expects cygwin-setup-x86_64.exe to be in the phoronix-test-suite directory.
**Lastly** the NASM repository always seems to be down and nasm test fails to install.  I grabbed nasm-2.13.01.tar.xz from a ftp repository and converted it to tar.bz2 since I had no luck changing the proper script to reflect the original file extension.  Then I placed the nasm tar.bz2 file into the installed tests directory /.phoronix-test-suite/pts/namd-1.0.1/  and reran pts/cpu installation to pickup that file.

All tests in pts/cpu suite is now working!  Commenting here in case someone else needs the info.  This problem and solution probably needs its own issue number for bug tracking.
